packages/components/pie-chip/src/index.ts

```diff
@@ -61,6 +64,8 @@ export class PieChip extends LitElement implements ChipProps {
     private renderCloseButton (): TemplateResult {
         return html`
                     <button
+                        @click="${() => dispatchCustomEvent(this, ON_CHIP_CLOSE_EVENT)}"
```

Comment:
I did it at first but since the function only dispatches the event it felt like waste of space 😄 
```
    private _handleClick () : void {
        dispatchCustomEvent(this, ON_CHIP_CLOSE_EVENT);
    }
````
I don't mind adding it if you see the value in doing so as I don't have a strong preference about it